ansible/roles/test/tasks/neighbour-mac-noptf.yml

```diff
@@ -80,7 +80,7 @@
     assert: { that: "arptable.v4[v4_nei]['macaddress'] | lower ==  v4_mac2" }
 
   - name: Get the SONiC Asic DB key holding neighbour MAC for {{ v4_nei }}
-    shell: docker exec database redis-cli -n 1 KEYS ASIC_STATE:SAI_OBJECT_TYPE_NEIGHBOR_ENTRY* | grep {{ v4_nei }}
+    shell: docker exec database redis-cli -n 1 KEYS ASIC_STATE:SAI_OBJECT_TYPE_NEIGHBOR_ENTRY* | grep '\"{{ v4_nei }}\"'
```

Comment:
>grep [](start = 97, length = 4)

Use exact fix-string match instead of regex match, because there is '.' in IP string.
```
 grep -Fe 
```